WordPress/src/main/res/values/key_strings.xml

```diff
@@ -165,6 +165,18 @@
         <item>2000</item>
     </string-array>
 
+    <!-- App Settings - Editor -->
+    <string-array name="editor_values">
```

Comment:
We're missing `translatable="false"` here and in the other array below (sorry I didn't spot that one in previous PR).